### PR TITLE
Cb 7 features

### PR DESCRIPTION
**Pull request by kNgo8249**
* **Please check if the PR fulfills these requirements**
- [:heavy_check_mark:] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
New feature (a "features" section)

* **What is the current behavior?What does the code do before you made the new changes?** 
Beforehand, the website did not have a features section

* **What is the new behavior?**
Now the website has a features component

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
There is no breaking change

* **Other information**:

### CODE REVIEW

File: frontend/src/components/LandingPage/Features/Features.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    Card, CardTitle, CardText, CardDeck,
+    CardBody, CardSubtitle
+  } from 'reactstrap';
+  
+function Features() {
+    return (
+        <div>
+            <h1 className="text-center">Features of our Application</h1>
+            
+            <CardDeck>
```
Comment (Codubee) [inline frontend/src/components/LandingPage/Features/Features.js:12]: What made you go with a CardDeck? I have never used it myself, just wondering what led you to using it. Not a critique just asking out of curiosity.

**Review by Codubee — APPROVED** [review]
Great work Kenny! This looks good to me

File: frontend/src/components/LandingPage/Features/Features.js (hunk 1 above)
Comment (kNgo8249) [reply frontend/src/components/LandingPage/Features/Features.js:12]: I wanted to use several cards for the features section as that's normally how I see features for something listed on a website. I was going to use a CardGroup but CardGroup has all the cards stuck together, and I wanted the cards to be each separate entity, and CardDeck does that.

File: frontend/src/components/LandingPage/Features/Features.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }); 
+
+    it('Check if cards render text properly', () => {
+        render(<Features/>);
+        let cardTitles = [/Guess Future Stock Prices/i, /Get Stock Information/i, /Get Current News/i]
+        cardTitles.forEach(element => {
+            const textElement = screen.getByText(element);
+            expect(textElement).toBeInTheDocument();
+        });
+    });
```
Comment (PretaxEnd9716) [inline frontend/src/components/LandingPage/Features/Features.test.js]: The test seems good, but I think you can make it even better by adding all the text of each card. You don't have to test the whole paragraph, but just a little snippet of it.

**Review by PretaxEnd9716 — COMMENTED** [review]
The changes seems good overall. Good work on the test cases!

**Review by ryannd — APPROVED** [review]
Looks good to me! Good job

**Pull request merged**